### ISSUE TITLE
PHP 8.1 `enum` not recognized

### ISSUE THREAD

**Issue by fefas** [squizlabs/PHP_CodeSniffer#3474]
**Describe the bug**
PHP 8.1 `enum` not recognized.

**Code sample**
```php
<?php

enum MyEnum
{
    case CONST1;
    case CONST2;
}
```

**To reproduce**

Just run the PHPCS with the `PSR12` ruleset and the output will be something like:
```
FILE: /var/www/code/src/MyEnum.php
-----------------------------------------------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 3 LINES
-----------------------------------------------------------------------------------------------------------------------
 5 | ERROR | [x] Line indented incorrectly; expected 0 spaces, found 4
   |       |     (Generic.WhiteSpace.ScopeIndent.IncorrectExact)
 6 | ERROR | [x] Line indented incorrectly; expected 0 spaces, found 4
   |       |     (Generic.WhiteSpace.ScopeIndent.IncorrectExact)
 7 | ERROR | [x] Line indented incorrectly; expected at least 4 spaces, found 0
   |       |     (Generic.WhiteSpace.ScopeIndent.Incorrect)
-----------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------------------------------------------
```

By running the auto fix, I get:

```php
<?php

enum MyEnum
{
case CONST1;
case CONST2;
    }
```

**Expected behavior**
The following code should cause no CS error:
```php
<?php

enum MyEnum
{
    case CONST1;
    case CONST2;
}
```

**Versions (please complete the following information):**
 - OS: Docker on MacOS (image tag `php:8.1.0RC5-cli-alpine3.14`)
 - PHP: 8.1 RC5 
 - PHPCS: 3.6.1
 - Standard: PSR12

**Comment by jrfnl** [comment]
PHPCS currently doesn't contain any syntax support yet for new syntaxes added in PHP 8.1. Work on this still has to start as all maintainers are overloaded.

**Comment by fefas** [comment]
No pressure :) just reporting and trying to contribute somehow

**Comment by kukulich** [comment]
I plan to do something to PHP 8.1 support this weekend.

**Comment by iansltx** [comment]
To clean this up, ScopeIndentSniff will need to be updated as well, looks like.

**Comment by michnovka** [comment]
I still have thi issue with 3.6.2. Was it fixed or not?

**Comment by derrabus** [comment]
The fix hasn't been released yet.

**Comment by Magiczne** [comment]
Is there any update on when this will be released?

**Comment by jrfnl** [comment]
@Magiczne When the release is ready.

**Comment by TheWildHorse** [comment]
@jrfnl Any updates on this?

**Comment by wesselstam** [comment]
@koficoud  It was released 2 hours ago :).

Thanks all!
